src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,10 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap);
```

Comment:
Maybe it's a better way to use a tag 'link' in the tag 'head' for fonts rather than import it in the scss file.

Reply:
@bohdan-mykhailenko why u think so? Actually, maybe it's a bit better to link a font to html tag performance vise, but it really isn’t worth worrying too much about  in this case.